### DISCUSSION TITLE
[refactor] wg-easy backend: framework

### DISCUSSION THREAD

**Discussion by pheiduck** [wg-easy/wg-easy#890] [Polls]
h3: https://github.com/wg-easy/wg-easy/pull/880
koa: https://github.com/wg-easy/wg-easy/pull/794

**Comment by cany748** [comment]
The advantages of `h3` over `koa`:

1. Written with Typescript and ESM.
2. Actively developing and updating.
3. Express [compatible](https://h3.unjs.io/examples/from-expressjs-to-h3).
4. Supports [more than just](https://h3.unjs.io/adapters) Node.js.
5. Faster and smaller.
6. Has a great ecosystem in the form of [UnJS](https://unjs.io/)

**Comment by pheiduck** [comment]
Thanks folks for your Votes! We choose H3 for our next Release v13.
